cfgov/templates/find_a_housing_counselor.html

```diff
@@ -5,6 +5,12 @@
 Find a housing counselor
 {% endblock %}
 
+{% block analytics_js %}
+<!-- Google Optimize --> 
```

Comment:
Sure -- I was thinking performance-wise it might be better to just load the Optimize code, but I can add that if we're not ready to remove the analytics code from this page yet.